hasura/migrations/default/1653331923087_add_comments_on_tables/down.sql

```diff
@@ -0,0 +1,8 @@
+
+comment on TABLE "public"."profiles" is E'NULL';
```

Comment:
are these comments visible in our graphql explorer? if so, maybe we should add some content here while going through effort of making this migration?

Reply:
This is a down migration for rolling back the migration that adds comments to these tables (`down.sql` vs `up.sql`). When set to null it wont show anything in the explorer, just like most of the tables right now.

You are right though that we should have comments on all our tables to document our API better. Happy to go do a pass of that after this work

Reply:
oops - didn't realize this was a down.